practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>"please refer to the <a href="https://github.com/engmubarak48/indaba-pracs-2024/blob/recommender-systems/practicals/generate_graph.py" target="_blank">generate_graph.py</a>" -&gt; "please refer to the file <a href="https://github.com/engmubarak48/indaba-pracs-2024/blob/recommender-systems/practicals/generate_graph.py" target="_blank">generate_graph.py</a>"</p><p> </p><p>last year -&gt; last year's</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>